handlers/repo/update.js

```diff
@@ -72,15 +72,24 @@ export const command = async ({ member, respond, content, channel, guild }) => {
     file: decodeURI(url).split("/").pop().split("?")[0],
     hash: createHash("md5").update(new Uint8Array(modFile)).digest("hex"),
   };
+
   const mods = await getTrackedData(
-    "https://raw.githubusercontent.com/SkyblockClient/SkyblockClient-REPO/main/files/" + (isBeta ? "mods_beta.json" : "mods.json")
+    "https://raw.githubusercontent.com/SkyblockClient/SkyblockClient-REPO/main/files/mods.json"
   );
-  const existingMod = mods.find((mod) => mod.forge_id == modId);
-  if (!existingMod) {
+  
+  const modsBeta = isBeta ? await getTrackedData(
+    "https://raw.githubusercontent.com/SkyblockClient/SkyblockClient-REPO/main/files/mods_beta.json"
+  ) : mods;
+
+  if (!mods.find((mod) => mod.forge_id == modId)) {
```

Comment:
Even if this now bypasses the issue, it won't be able to apply the update in the final stage if you don't update updateCheck3